### DISCUSSION TITLE
Request: Create a new standalone note by shortcut

### DISCUSSION THREAD

**Discussion by ulahcherubim** [windingwind/zotero-actions-tags#271] [Ideas]
Hi, I tried to get the useful information for writing this, but couldn't get my hands on what are the names of buttons/functions to create a new note. Any help would be appreciated!

**Comment by windingwind** [comment]
https://github.com/zotero/zotero/blob/a7d59a90d37695e4eaa0e42d9e2301b049d0a0e7/chrome/content/zotero/zoteroPane.js#L4286

**Reply by ulahcherubim** [reply]
Thank you, will give it a try!
